org.adempiere.base/src/org/compiere/model/MArchive.java

```diff
@@ -402,12 +406,26 @@ public File saveAsZip() {
 	 * @param Record_ID
 	 * @param trxName
 	 * @return int[], [0] = report count and [1] = document count
+	 * @deprecated - use {@link #getReportAndDocumentCountByRecordId(int, int, String, String)} instead
 	 */
 	public static int[] getReportAndDocumentCountByRecordId(int AD_Table_ID, int Record_ID, String trxName) {
+		return getReportAndDocumentCountByRecordId(AD_Table_ID, Record_ID, null, trxName);

```

Comment:
this couple with the Record_UU changes below is not just deprecation but it is a breaking change.

Reply:
Thanks, fixed with commit https://github.com/idempiere/idempiere/pull/1895/commits/1654f4595e835024cc3467d2148e6034dff5d4f5

Reply:
hi @CarlosRuiz-globalqss , with the removal of OR, this method is not going to work.